src/Controllers/LoggerController.js

```diff
@@ -4,6 +4,8 @@ import AdaptableController from './AdaptableController';
 import { LoggerAdapter } from '../Adapters/Logger/LoggerAdapter';
 
 const MILLISECONDS_IN_A_DAY = 24 * 60 * 60 * 1000;
+const LOG_STRING_TRUCATE_LENGTH = 1000;
```

Comment:
Can we make this configurable somehow? It's been useful so far in dev not to have this limit